pom.xml

```diff
@@ -270,6 +270,11 @@ SOFTWARE.
       <artifactId>handlebars</artifactId>
       <version>4.2.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.javatuples</groupId>
+      <artifactId>javatuples</artifactId>
+      <version>1.2</version>
+    </dependency>
```

Comment:
@Sammers21 please avoid using this library. Reading code with tuples is hard. Creating custom class of finding other solution is preferred